HHS/Head-Start-TTADP
# [Production] Merge goals front-end, email notifications for training report, update approver display

## Description of change

### 1938
Implement email notifications for TR and session reports. Also, disable the "submit session" button for a POC

### 633
List approvers and add icons to the TR approval chart

### 1136
Create the front-end form for merge goals

## How to test

For 1938, consult the Jira for the individual emails. To confirm emails are being sent, we use impersonation to trigger a send and then consult the logs for confirmation. Also confirm that the "submit session" button is disabled for a POC.

For 633, again user impersonation will be key, but create a report with two approvers and then go through the individual process of approving and marking a report needs action, respectively. Confirm the alerts and the table show the correct information.

For 1136, view the RTR with a feature flag or as an admin. On any existing recipient with goals, you'll see that the merge goals "warning banner" appears. Click through the form, paying attention to validation as you go. (Note that "submit" doesn't currently do anything)

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-633
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1938
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1136

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
